test/models/vote_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+describe Vote do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+
+  describe 'validations' do
+    before do
+      @vote = Vote.new(user_id: 1, work_id: 1)
```

Comment:
This won't work because you can't guarantee that the test database will have a user and work with id 1.

Either:

1. Create fixtures for your test database and use them or
1. Do something like this

```suggestion
      @user = User.create(username: 'bob');
      @user = User.create(username: 'bob');
      @work = Work.create(category: 'book', title: 'Wizard of Oz', creator: 'Someone', publication_year: "1920", description: 'a book about a girl who got hit on the head')
      @vote = Vote.new(user_id: @user.id, work_id: @work.id)
```